File: submissions/Morbid Mask/problem1.java
import java.util.*;
class problem1{
public static void main(String arg[]){
int r,b=0;
System.out.println("Enter an integer number:");
Scanner sc=new Scanner(System.in);
int n=sc.nextInt();
q=n;
while(q!=0){
r=q%10;
int b=b+fact(r);
q=q/10;
}

static int fact(int k){
int f=1;
for(int i=k;i>0;i--){
f=f*i;
}
return f;
}
}
}

File: submissions/devil/problem1.cpp
#include<stdio.h>
int main()
{
	int fact,x,d,sum=0,a,n;
printf("Enter a number");
scanf("%d",&a);

fact=1;

 while(a>0)
   {
   	d=n%10;
   	for(x=1;x<=d;x++)
   	  {
   	  	fact=fact*x;
		 }
		 sum=sum+fact;
		 n=n/10;
   }
if(sum==a)
printf("KRISNAMURTHY NO");
else 
printf("not a KRISNAMURTHY NO");
}
